docs/02.API-REFERENCE.md

```diff
@@ -6988,7 +6988,7 @@ jerry_error_sz (jerry_error_t error_type, const char *message_p);
 Create new JavaScript DataView object.
 
 *Note*:
-  - This API depends on the es.next profile.
+  - This API depends on a build option (`JERRY_BUILTIN_DATAVIEW`) and can be checked
```

Comment:
This sentence seems unfinished to me. `and can be checked with [jerry_feature_enabled](#jerry_feature_enabled).`